store/tikv/range_task.go

```diff
@@ -168,7 +166,12 @@ func (s *RangeTaskRunner) RunOnRange(ctx context.Context, startKey []byte, endKe
 		}
 
 		pushTaskStartTime := time.Now()
-		taskCh <- task
+
+		select {
+		case taskCh <- task:
+		case <-ctx.Done():
+			break
```

Comment:
I think need return err here.

Reply:
It should run to line https://github.com/pingcap/tidb/pull/10622/files#diff-8a388eccc19df74aa4991690c5f508a5L192 to return error!

Reply:
got it :) LGTM!